modules/phase_field/src/userobjects/DiscreteNucleationFromFile.C

```diff
@@ -90,9 +111,18 @@ DiscreteNucleationFromFile::DiscreteNucleationFromFile(const InputParameters & p
     paramError("file", "Missing 'y' column in file");
   if (!found_z && _mesh.dimension() >= 3)
     paramError("file", "Missing 'z' column in file");
+  if (!found_r && !_fixed_radius)
+    paramError("file", "Missing 'r' column in file");
+  if (found_r && _fixed_radius)
```

Comment:
Let's not error out here. It seems perfectly valid to want to override the radius data in a CSV file and substitute a fixed radius for it.

Reply:
I think you're referring to lines 116 - 117 there, remove the check and the error for 
`if (found_r && _fixed_radius)`, right?